pkg/apis/kops/v1alpha1/conversion.go

```diff
@@ -0,0 +1,241 @@
+// +build !ignore_autogenerated
+
```

Comment:
Actually even better... this file is _not_ autogenerated, and we already name the autogen files `zz_generated`.  Will fix this one :-)